### PR TITLE
[E2E Relay, Configs] Fix localnet E2E relay issue - Ready for Tilt Integration

### PR DESCRIPTION
**Pull request by bryanchriswhite**
#### To run the E2E relay test outside of tilt:

```bash
# Start the appgate-server (blocks terminal)
poktrolld --home ./localnet/poktrolld appgate-server --keyring-backend test --config ./localnet/poktrolld/config/appgate_server_config.yaml

# Start the relayminer (blocks terminal)
poktrolld --home ./localnet/poktrolld/ relayminer --keyring-backend test  --config ./localnet/poktrolld/config/relayminer_config.yaml

# Stake supplier2 & app2 so their keys are on-chain
make supplier2_stake && make app2_stake

# Use curl to send a relay request via the appgate-server
 curl -v POST -H "Content-Type: application/json" --data \
    '{"jsonrpc":"2.0","method":"eth_blockNumber","id":1}' \ # NOTE: the addition of the "id" property.
    http://localhost:42069/anvil

# Run just the relay feature file
go test -v -tags=e2e ./e2e/tests  -features-path=./relay.feature
```

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [x] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Run all unit tests**: `make go_develop_and_test`
- [ ] **Verify Localnet manually**: See the instructions [here](TODO: add link to instructions)

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, updated documentation and left TODOs throughout the codebase

### CODE REVIEW

File: localnet/poktrolld/config/relayminer_config.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # tcp://<host>:<port> to a full pocket node for reading data and listening for on-chain events
-query_node_url: tcp://localhost:26657
+query_node_url: tcp://localhost:36657
 # tcp://<host>:<port> to a pocket node that gossips transactions throughout the network (may or may not be the sequencer)
 network_node_url: tcp://127.0.0.1:36657
 # Name of the key (in the keyring) to sign transactions
-signing_key_name: servicer1
+signing_key_name: supplier2
```
Comment (bryanchriswhite) [inline localnet/poktrolld/config/relayminer_config.yaml:6]: **BEFORE MERGE**: replace with `supplier1`

File: localnet/poktrolld/config/relayminer_config.yaml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 # TODO_TECHDEBT(#137, #130): Once the `relayer.json` config file is implemented AND a local LLM RPC service
 # is supported on LocalNet, this needs to be expanded to include more than one service. The ability to support
 # multiple services is already in place but currently (as seen below) is hardcoded.
 # TODO_UPNEXT(@okdas): this hostname should be updated to match that of the in-tilt anvil service.
 proxied_service_endpoints:
-  anvil: http://anvil:8080
+  anvil: http://localhost:8080
```
Comment (bryanchriswhite) [inline localnet/poktrolld/config/relayminer_config.yaml:12]: **BEFORE MERGE**:
```suggestion
  anvil: http://anvil:8080
```

File: localnet/poktrolld/config/supplier2_stake_config.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 - service_id: anvil
   endpoints:
-  - url: http://anvil:8547
+  - url: http://localhost:8545
```
Comment (bryanchriswhite) [inline localnet/poktrolld/config/supplier2_stake_config.yaml:3]: **BEFORE MERGE**:

```suggestion
  - url: http://anvil:8545
```

**Review by red-0ne — APPROVED** [review]
LGTM :+1:

**Pull request merged**